contrib/endpoints/src/api_manager/context/service_context.h

```diff
@@ -65,8 +65,12 @@ class ServiceContext {
     return !is_auth_force_disabled_ && config_->HasAuth();
   }
 
-  bool RequireRulesCheck() const {
-    return RequireAuth() && !is_check_security_rules_disabled_;
+  bool IsRulesCheckEnabled() const {
+    return RequireAuth() && config_->server_config() &&
+           !config_->server_config()
+                ->api_check_security_rules_config()
+                .firebase_server()
```

Comment:
Added it. For some reason this is not being done in other parts of the code. I am adding this check in one place I know.